### PR TITLE
chore: switch from jsonb to jackson

### PR DESCRIPTION
**Pull request by dvince2**
# Changing serialization from reasteasy jsonb to fasterxml jackson

* In theory this change should be seamless and transparent.  I've tested as many endpoints and workflows as I could, however, there are some preexisting issues which stop a complete test.
* This can be thought of as a 'stepping stone' to allow us to use Springboot easier.

### CODE REVIEW

File: packages/services/hmi-server/src/main/java/software/uncharted/terarium/hmiserver/models/dataservice/Dataset.java [hunk 1]
```diff
@@ -25,16 +27,17 @@ public class Dataset extends ResourceType implements Serializable {
...
+	@JsonProperty("temporal_resolution")
 	private String temporalResolution;
 
-	@JsonbProperty("geospatial_resolution")
+	@JsonProperty("geospatial_resolution")
 	private String geospatialResolution;
 
-	private String annotations;
+	//TODO not an object?
+	private Object annotations;
```
Comment (YohannParis) [inline packages/services/hmi-server/src/main/java/software/uncharted/terarium/hmiserver/models/dataservice/Dataset.java:37]: Make an issue of it.

File: packages/services/hmi-server/src/main/java/software/uncharted/terarium/hmiserver/models/dataservice/SimulationPlan.java [hunk 2]
```diff
@@ -1,23 +1,25 @@
...
+
+	@JsonProperty("model_id")
 	private String modelId;
 
 	private String simulator;
 
 	private String query;
 
-	private String content;
+	private Object content;
```
Comment (YohannParis) [inline packages/services/hmi-server/src/main/java/software/uncharted/terarium/hmiserver/models/dataservice/SimulationPlan.java:22]: Same issue as above

File: packages/services/hmi-server/src/main/java/software/uncharted/terarium/hmiserver/models/documentservice/ExtractionProperties.java [hunk 3]
```diff
@@ -29,8 +31,8 @@ public class ExtractionProperties implements Serializable {
 	private String contentText;
 
 	private Number indexInDocument;
-
-	private Map<String, Map<String, String>> contentJSON;
+	//TODO fixme from object
+	private Object contentJSON;
```
Comment (YohannParis) [inline packages/services/hmi-server/src/main/java/software/uncharted/terarium/hmiserver/models/documentservice/ExtractionProperties.java:35]: Same issue above

**Review by YohannParis — APPROVED** [review]
LGTM

File: packages/services/hmi-server/src/main/java/software/uncharted/terarium/hmiserver/models/dataservice/Dataset.java (hunk 1 above)
Comment (dvince2) [reply packages/services/hmi-server/src/main/java/software/uncharted/terarium/hmiserver/models/dataservice/Dataset.java:37]: https://github.com/DARPA-ASKEM/Terarium/issues/818

File: packages/services/hmi-server/src/main/java/software/uncharted/terarium/hmiserver/models/documentservice/ExtractionProperties.java (hunk 3 above)
Comment (dvince2) [reply packages/services/hmi-server/src/main/java/software/uncharted/terarium/hmiserver/models/documentservice/ExtractionProperties.java:35]: https://github.com/DARPA-ASKEM/Terarium/issues/818

File: packages/services/hmi-server/src/main/java/software/uncharted/terarium/hmiserver/models/dataservice/SimulationPlan.java (hunk 2 above)
Comment (dvince2) [reply packages/services/hmi-server/src/main/java/software/uncharted/terarium/hmiserver/models/dataservice/SimulationPlan.java:22]: https://github.com/DARPA-ASKEM/Terarium/issues/818

**Review by YohannParis — APPROVED** [review]

**Pull request merged**